packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -420,6 +427,14 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     return torrent;
   }
 
+  protected async closeChannels(torrent: PaidStreamingTorrent) {
+    torrent.wires.forEach(async wire => {
+      if (wire.paidStreamingExtension && wire.paidStreamingExtension.peerChannelId) {
+        await this.paymentChannelClient.closeChannel(wire.paidStreamingExtension.peerChannelId);
+      }
+    });
+  }
+
```

Comment:
Can we add a comment here explaining that we only close channels that our peer opened (i.e. that we are paying *out* in) ?

We might even want to rename `closeDownloadingChannels` or something like that. 

Obviously this behaviour means that I can still get paid for uploading the bits of the file I still have.

Reply:
fixed.